src/ops/python_compat.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Adobe. All rights reserved.
```

Comment:
Why not use six (https://six.readthedocs.io/) for compatibility?

Reply:
Done